pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -186,3 +210,39 @@ func (c *openServiceBrokerClient) DeleteServiceBinding(sID, bID string) error {
 
 	return nil
 }
+
+func (c *openServiceBrokerClient) pollBroker(ID string, operation string) error {
+	pollReq := brokerapi.LastOperationRequest{}
+	if operation != "" {
+		pollReq.Operation = operation
+	}
+
+	url := fmt.Sprintf(pollingFormatString, c.broker.Spec.URL, ID)
+	for i := 0; i < pollingAmountLimit; i++ {
+		glog.Infof("Polling attempt #%v: %s", i+1, url)
+		pollResp, err := util.SendRequest(c.client, http.MethodGet, url, pollReq)
+		if err != nil {
+			return err
+		}
+
+		lo := brokerapi.LastOperation{}
+		if err := util.ResponseBodyToObject(pollResp, &lo); err != nil {
+			return err
+		}
+
+		switch lo.State {
+		case "in progress":
+		case "succeeded":
+			return nil
+		case "failed":
```

Comment:
From the spec https://docs.cloudfoundry.org/services/api.html#polling

Valid values for state are `in progress`, `succeeded`, and `failed`. The platform will poll the last_operation endpoint as long as the broker returns "state": "in progress". Returning "state": "succeeded" or "state": "failed" will cause the platform to cease polling.